<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SatelliteAvoidanceWidget</class>
 <widget class="QWidget" name="SatelliteAvoidanceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="2" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>error margin</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>orbit error</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>orbit error / day</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>min elevation</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>check frequency</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="spinBox_orbitError">
         <property name="toolTip">
          <string>orbital error perpenticular to observation direction</string>
         </property>
         <property name="statusTip">
          <string>orbital error perpenticular to observation direction</string>
         </property>
         <property name="whatsThis">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;orbital error at TLE epoch perpenticular to observation direction. &lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;Used to calculate the avoidance area around satellites. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="suffix">
          <string> [m]</string>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
         <property name="singleStep">
          <number>500</number>
         </property>
         <property name="value">
          <number>2000</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="spinBox_orbitErrorPerDay">
         <property name="toolTip">
          <string>increase in orbital error perpenticular to observation direction per day</string>
         </property>
         <property name="statusTip">
          <string>increase in orbital error perpenticular to observation direction per day</string>
         </property>
         <property name="whatsThis">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;increase in orbital error perpenticular to observation direction per day.&lt;/p&gt;&lt;p&gt;Used to calculate the avoidance area around satellites&lt;br/&gt;&lt;/p&gt;&lt;p&gt;E.g. &lt;/p&gt;&lt;p&gt;orbital error = 2 km&lt;/p&gt;&lt;p&gt;orbital error per day = 1 km &lt;/p&gt;&lt;p&gt;observation start is in 4.5 days&lt;/p&gt;&lt;p&gt;--&amp;gt; orbital error = 6.5 km &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="suffix">
          <string> [m/day]</string>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
         <property name="singleStep">
          <number>500</number>
         </property>
         <property name="value">
          <number>2000</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_errorMargin">
         <property name="toolTip">
          <string>additional angular distance around satellites that is avoided</string>
         </property>
         <property name="statusTip">
          <string>additional angular distance around satellites that is avoided</string>
         </property>
         <property name="whatsThis">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;additional error margin that is added on top of orbital error.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;E.g. &lt;/p&gt;&lt;p&gt;avoidance because of orbital error = 2.4 degrees &lt;/p&gt;&lt;p&gt;error margin = 0.2 degrees&lt;/p&gt;&lt;p&gt;total area around satellite that will be avoided during observations = 2.6 degrees &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="suffix">
          <string> [deg]</string>
         </property>
         <property name="maximum">
          <double>180.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>0.200000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_minElevation">
         <property name="toolTip">
          <string>minimum elevation of satellites to be avoided</string>
         </property>
         <property name="statusTip">
          <string>minimum elevation of satellites to be avoided</string>
         </property>
         <property name="whatsThis">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;minimum elevation of satellites to be avoided. &lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;E.g. &lt;/p&gt;&lt;p&gt;Satellites with lower elevation are not avoided because it is assumed that they will not propagate signal in the direction of the receiver &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="suffix">
          <string> [deg]</string>
         </property>
         <property name="maximum">
          <double>90.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>5.000000000000000</double>
         </property>
         <property name="value">
          <double>20.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSpinBox" name="spinBox_checkFrequency">
         <property name="toolTip">
          <string>frequence in which satellite positions are compared with observation angles</string>
         </property>
         <property name="statusTip">
          <string>frequence in which satellite positions are compared with observation angles</string>
         </property>
         <property name="whatsThis">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;frequence in which satellite positions are compared with observation angles.&lt;/p&gt;&lt;p&gt;Satellite position will always be checked at the start end end of the observation. &lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;E.g. &lt;/p&gt;&lt;p&gt;check frequency = 10 sec&lt;/p&gt;&lt;p&gt;observation start = 00:00:30 &lt;/p&gt;&lt;p&gt;observation end = 00:01:05&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;Satellite position is evaluated at &lt;/p&gt;&lt;p&gt;- 00:00:30&lt;/p&gt;&lt;p&gt;- 00:00:40&lt;/p&gt;&lt;p&gt;- 00:00:50&lt;/p&gt;&lt;p&gt;- 00:01:00&lt;/p&gt;&lt;p&gt;- 00:01:05&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="suffix">
          <string> [s]</string>
         </property>
         <property name="maximum">
          <number>600</number>
         </property>
         <property name="singleStep">
          <number>5</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_out">
         <property name="toolTip">
          <string>output progress report for preprocessing to log file</string>
         </property>
         <property name="statusTip">
          <string>output progress report for preprocessing to log file</string>
         </property>
         <property name="whatsThis">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;At the start of the scheuduling, VieSched++ will actively search for time periods were satellites are visible by the station. &lt;/p&gt;&lt;p&gt;This is necessary to significantly speed up the actual scheduling. &lt;/p&gt;&lt;p&gt;However, these pre-processings can take a time (VieSched++ will check visibility for every station + satellite pair every 60 seconds which can easily accumulate to several thousand tests)&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;With this option, you can output the progress in these calculations to the .log file. &lt;/p&gt;&lt;p&gt;They will be outputted as [info] messages. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="prefix">
          <string>every </string>
         </property>
         <property name="suffix">
          <string> %</string>
         </property>
         <property name="minimum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>5.000000000000000</double>
         </property>
         <property name="value">
          <double>10.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QCheckBox" name="checkBox_out">
         <property name="toolTip">
          <string>output progress report for preprocessing to log file</string>
         </property>
         <property name="statusTip">
          <string>output progress report for preprocessing to log file</string>
         </property>
         <property name="whatsThis">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;At the start of the scheuduling, VieSched++ will actively search for time periods were satellites are visible by the station. &lt;/p&gt;&lt;p&gt;This is necessary to significantly speed up the actual scheduling. &lt;/p&gt;&lt;p&gt;However, these pre-processings can take a time (VieSched++ will check visibility for every station + satellite pair every 60 seconds which can easily accumulate to several thousand tests)&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;With this option, you can output the progress in these calculations to the .log file. &lt;/p&gt;&lt;p&gt;They will be outputted as [info] messages. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>output to log</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_save">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../myresources.qrc">
           <normaloff>:/icons/icons/document-export.png</normaloff>:/icons/icons/document-export.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>397</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../myresources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>checkBox_out</sender>
   <signal>toggled(bool)</signal>
   <receiver>doubleSpinBox_out</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
